code/index.html

```diff
@@ -3,28 +3,275 @@
+      </div>
+
+      <div class="question" id="question3">
+        <h4>Finns det bastu?</h4>
+      </div>
+      <div class="answer-container">
+        <p class="answer">I alla simhallar finns det vanlig bastu, i vissa finns även ångbastu.</p>
+      </div>
+
+      <div class="question" id="question4">
+        <h4>Måste jag duscha innan jag badar?</h4>
+      </div>
+      <div class="answer-container">
+        <p class="answer">Du ska alltid duscha utan badkläder. Vi har valt tvål med omsorg, så att duschen blir en del av din simritual.</p>
+      </div>
+
+      <div class="question" id="question5">
+        <h4>Kan jag ta med mig barnen?</h4>
+      </div>
+      <div class="answer-container">
+        <p class="answer">Absolut. Under vissa tider är det lek i bassängerna och vi har även 
+          <a href="#">simskolor</a> för de små.
+        </p>
+      </div>
+  </section>
+
+<!-- Javascript file! -->
+  <script src="./script.js"></script>
+
+<!-- Article! -->
+  <article class="card-container">
+    <div class="card-right">
+      <img
+        src="https://motionera.stockholm/optimized/c2x1_xxlarge/globalassets/motionera-simma-och-trana/simhallar/akeshovssimhall.jpg" 
+        class="card-img" 
+        alt="Pool"
+      >
+    <div class="text-container-right">
+      <h2>Missa inte våra simkurser i Åkeshov</h2>
+        <p class="card-text">Just nu har vi platser kvar på vår simkurser i Åkeshovs simhall. Säkra en plats redan idag. Kursen pågår mellan 19.30-21.00 varje måndag kväll. Start den 20 september och under 8 gånger framåt.
+          <li>Crawlkurs</li>
+          <li>Fjärilssim</li>
+          <li>Konstsim</li>
+        </p>
+      </div>
+    </div>
+  </article>
 
-    <section class="accordion">
-      <!-- Accordion -->
-      <h1>FAQ</h1>
+<!-- Form -->
+  <form 
+      class="form-container" 
+      action="http://httpbin.org/anything" 
+      method="POST">
 
-      <div class="question" id="section1">Section 1</div>
-      <div class="answer">
-        <p>Lorem ipsum...</p>
+    <div class="form1">
+
+      <h3 class="form-heading">Säkra en plats direkt</h3>
+        <p class="form-intro"> 
+          Har du redan ett inlogg? Då kan du 
+          <a class="intro-link" href="#">logga in</a> 
+          så slipper du ange dina uppgifter igen.
+        </p>
+        <h4>Välj kurs</h4>
+
+      <div class="form-group">
+        <label for="kurs">Crawlkurs
+        <input 
+          id="kurs1" 
+          type="radio" 
+          value="courseValue"
+        />
+        </label>
+      </div>
+
+      <div class="form-group">
+        <label for="kurs">Fjärilssim
+        <input 
+          id="kurs2" 
+          type="radio" 
+          name="courseValue"
+        />
+        </label>
+      </div>
+      
+      <div class="form-group">
+        <label for="kurs">Konstsim
+        <input 
+          id="kurs3" 
+          type="radio" 
+          name="courseValue"
+        />
+        </label>
       </div>
```

Comment:
Your inputs' arguments are differing. They should all have a `value` and a `name`. This creates a little bug (it's possible to check more than one radio button). Update this so that all of them have different values (the first one could have the value crawl so it reflects the user's choice), and the name could stay as `courseValue`. A nice feature when working with radio buttons in to make the first radio button checked as default, like this:

```
<input 
  checked
  id="kurs1" 
  type="radio" 
  value="crawl"
  name="courseValue"
/>
```